src/main/resources/db/migration/changelog-master.json

```diff
@@ -1,5 +1,8 @@
 { "databaseChangeLog": [
-    {
-        "includeAll": {"path": "db/migration/changes/"}
-    }
+        {"include": {"file": "db/migration/changes/Users.json"}},
+        {"include": {"file": "db/migration/changes/RecommendationRequest.json"}},
+        {"include": {"file": "db/migration/changes/RequestType.json"}},
+        {"include": {"file": "db/migration/changes/Restaurants.json"}},
+        {"include": {"file": "db/migration/changes/UCSBDates.json"}},
+        {"include": {"file": "db/migration/changes/UCSBDiningCommons.json"}}
```

Comment:
why?

Reply:
I mentioned it in the PR description but the change to changelog-master.json was necessary because otherwise we could not guarantee in the runall command that Users table would be setup before RecommendationRequest, thus breaking the foreign key relationship that we have built in